conf/development.ini

```diff
@@ -1,6 +1,7 @@
 [app:main]
 use = call:lms.app:create_app
 debug = true
+dev = true
```

Comment:
Added a new "dev" config setting so that we know when we're in dev mode. It's true only in `make dev`, not on QA, in prod, or in tests,